src/middlewared/middlewared/plugins/pool_/replace_disk.py

```diff
@@ -16,6 +17,7 @@ class PoolService(Service):
         Str('disk', required=True),
         Bool('force', default=False),
         Str('passphrase', private=True),
+        Bool('preserve_settings', default=False),
```

Comment:
I think this should probably be True?

Reply:
So we change default behavior? I've thought we'll add checkbox in UI and let user decide.

Reply:
It was not a default behavior, it was a missing feature, thats the expected behavior.